twisted/conch/test/loopback.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Twisted Matrix Laboratories.
```

Comment:
can we have it twisted.test_proto_helpers ? it looks like a non conch specific helper